lib/ansible/plugins/filter/core.py

```diff
@@ -600,6 +610,8 @@ def filters(self):
             'win_basename': partial(unicode_wrap, ntpath.basename),
             'win_dirname': partial(unicode_wrap, ntpath.dirname),
             'win_splitdrive': partial(unicode_wrap, ntpath.splitdrive),
+            'commonpath': commonpath,
+            'normpath': partial(unicode_wrap, os.path.normpath),
```

Comment:
Why is only one of these wrapped?

Reply:
wrapped as in in the partial function? actually, for the `commonpath` we will need to provide a list of paths, and I can see that the `unicode_wrap` only returns a string as an output, thus I didn't wrap it.
@webknjaz Is there any better way to do that?